Nicolas

j314h opened the pull request.

Vaquicraft committed:
rollDice player + adv fonctionnel

Vaquicraft committed:
diceRoll sans design

j314h commented on html/index.html:
```diff
@@ -8,6 +8,21 @@
+        id="numberDices"
+        name="name"
+        type="number"
+        required
```
bien jouer le required

j314h commented on js/app.js:
```diff
@@ -0,0 +1,45 @@
+        startGame(nbDices)   
+    }
+
+    else {
```
merci pour tes efforts pour les accolades, mais les else se mette tout de suite apres l'accolade , encore un peu d'effort

j314h commented on js/app.js:
```diff
@@ -0,0 +1,45 @@
+    gameCount++
+}
+
+//Ca manquait de commentaires, alors j'en ai mis un ! :D
```
hahahahha et au moins il est pertinent hahaha

j314h commented on js/app.js:
```diff
@@ -0,0 +1,45 @@
+    }
+
+    else {
+        document.getElementById("player").innerHTML = ""
```
tu aurais pu faire mieux que inner dommage ....

Vaquicraft replied:
Effectivement, c'était un peu le joker de la fin, j'en ai discuté avant de partir et visiblement j'aurais pu faire un textContent à la place
J'ai quand même demandé à l'IA si c'était grave dans ce contexte, dans une problématique d'injection de code, mais comme c'est juste pour vider ça dérangeait pas, mais à l'avenir je ferai autrement
Mieux vaut un code qui fonctionne pas parfaitement comme attendu qu'un code rempli de failles de sécurité, j'imagine

j314h reviewed (commented):
tres tres bon code, aller je vais dire 1 erreur pour le inner, sinon parfait continu !!!!

Vaquicraft committed:
fix innerHTML > textContent + if pas logique

Inversion de la condition de vérif du nb de parties jouées

Remplacement du .innerHTML par un textContent pour vider les div

J'avais d'autres optis à faire, ça de démange mais je vais me retenir 😂

Vaquicraft committed:
myWorkHereIsDone

Vaquicraft committed:
le petit fix de la fin

Vaquicraft committed:
update
